rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+| Deshielded | Private State| Public State| Yes         | Transactions where the sender’s identity needs to be hidden.            | Ensures the sender’s identity remains confidential while making the transaction results public. Suitable for confidential disbursements or anonymized data publication.  |
+---
+
+
+By supporting a range of execution types, NSSA provides a flexible and robust framework for managing privacy and security in blockchain transactions. 
+Whether the need is for complete transparency, total privacy, or a balanced approach, NSSA's execution types allow users to select the level of confidentiality 
+that best fits their requirements. This flexibility enhances the overall utility of the blockchain, making it suitable for a wide array of applications and use cases.
+
+### e) Nescience Users
+---
+Nescience users are integral to the architecture, managing balances and assets within the blockchain network and invoking smart contracts with various privacy options. 
+They can choose the appropriate execution type—public, private, shielded, or deshielded—based on their specific privacy and security needs.
+
+Users handle both public and private balances. Public balances are visible to all network participants and suitable for non-sensitive transactions, 
+while private balances are confidential and used for transactions requiring privacy. Digital wallets provide a user-friendly interface for managing 
+these balances, assets, and transactions, allowing users to select the desired execution type seamlessly.
+
+Security is ensured through the use of cryptographic keys, which authenticate and verify transactions. Zero-Knowledge Proofs (ZKPs) maintain privacy 
+by validating transaction correctness without revealing underlying data, ensuring sensitive information remains confidential even during verification.
+
+The workflow for users involves initiating a transaction, preparing inputs, interacting with smart contracts, generating proofs if needed, 
+and submitting the transaction to the sequencer for verification and state update. This flexible approach supports various use cases, 
+from financial transactions and decentralized applications to data privacy management, allowing users to maintain control over their privacy settings.
+
+By offering this high degree of flexibility and security, Nescience enables users to tailor their privacy settings to their specific needs, 
+ensuring sensitive transactions remain confidential while non-sensitive ones are transparent. This integration of cryptographic keys and ZKPs 
+provides a robust framework for a wide range of blockchain applications, enhancing both utility and trust within the network.
+
+
+### f) Smart Contracts in Nescience
```

Comment:
Replace:
"Smart Contracts in Nescience"

With:
"Smart contracts in Nescience"